src/charm.py

```diff
@@ -461,7 +463,7 @@ def _on_add_admin_user_action(self, event: ActionEvent) -> None:
                 [
                     "bash",
                     "-c",
-                    f"./bin/bundle exec rake admin:create <<< $'{email}\n{password}\n{password}\nY'",
+                    f"./bin/bundle exec rake admin:create <<< $'{email}\n{password}\n{password}\nY'",  # pylint: disable=line-too-long
```

Comment:
I'm wondering if this is a case for using `format()` instead of a f-string with a pylint disable

Reply:
Good! Let me rework that, I don't like disabling errors when they can be fixed. Thanks!

Reply:
It is also possible to break this line into multiple lines and retaining f-strings, see other PR that had similar issues